packages/happner-2/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "happner-2",
-  "version": "12.9.0",
+  "version": "13.0.0",
```

Comment:
Do we want to jump a major - I mean, how many people updating custom_details or application_details?